resources/lang/en-US.yml

```diff
@@ -1677,4 +1677,9 @@ msg_err_could_not_find_any_towns_to_rtp: 'Could not find any open, public towns
 #Message showing payment for RTP.
 msg_cost_spawn_rtp: '&bYou were charged %s to teleport to a random town plot that is for sale.'
 #Message shown to a player when they use the town rtp command.
-msg_you_are_about_to_be_teleported_to_a_plot: 'You are about to be teleported to a plot which is forsale in %s which will let you join. Use ''/plot claim'' to claim your plot when you arrive.'
\ No newline at end of file
+msg_you_are_about_to_be_teleported_to_a_plot: 'You are about to be teleported to a plot which is forsale in %s which will let you join. Use ''/plot claim'' to claim your plot when you arrive.'
+
+#
+#Plots on Status Screen
+status_plot_string: 'Plots'
+status_plot_hover: '%s:'
```

Comment:
Each language string should have its own comment line, which is imported into Crowdin separately.